modin/config/__main__.py

```diff
@@ -62,12 +68,12 @@ def export_config_help(filename: str):
                 "Default Value": obj._get_default()
                 if obj.__name__ != "RayRedisPassword"
                 else "random string",
-                "Description": obj.__doc__,
+                "Description": description,
                 "Options": obj.choices,
             }
-            configs = configs.append(data, ignore_index=True)
+            configs_data.append(data)
 
-    configs.to_csv(filename, index=False)
+    pandas.DataFrame(configs_data).to_csv(filename, index=False)
```

Comment:
don't we want to pass column names here in order to preserve verbosity of the csv file?
```python
    configs = pandas.DataFrame(
        columns=[
            "Config Name",
            "Env. Variable Name",
            "Default Value",
            "Description",
            "Options",
        ]
    )
```

Reply:
Yes, sure. Columns names were added.